signac/__main__.py

```diff
@@ -1223,7 +1231,22 @@ def main():
         type=int,
         nargs='?',
         const=3,
-        help="Show the state point and document of each job.")
+        help="Show the state point and document of each job. Equivalent to "
+        "--sp NUM --doc NUM.")
+    parser_find.add_argument(
+        '--sp',
+        type=int,
+        nargs='?',
+        const=3,
+        help="Show the state point of each job. The parameter is the level of "
+        "nesting to display.")
```

Comment:
I think it would be better to use the argument to limit which state point key(s) are shown.
For example: `signac find --sp foo` would only show the `foo` state point, whereas `signac find --sp foo bar` shows both `foo` and `bar`. Both `--sp` and `--doc` would always imply `--show` and the `--show` argument would still be used to specify the nesting depth.

Reply:
If you specify `--doc` with some set of keys and `--show`, would you expect to get the full statepoint printed, or would you have to specify `--sp` as well in order to get it to print that? I'm not sure which API is more natural.

Reply:
Since it in its current form it would be impossible to say "do not show the state point at all", I would opt for the latter.

Reply:
OK that's consistent with how I've written the code right now.